staging/src/k8s.io/dynamic-resource-allocation/structured/allocator_test.go

```diff
@@ -240,13 +240,17 @@ func slice(name string, nodeSelection any, pool, driver string, devices ...resou
 	return slice
 }
 
-func deviceAllocationResult(request, driver, pool, device string) resourceapi.DeviceRequestAllocationResult {
-	return resourceapi.DeviceRequestAllocationResult{
+func deviceAllocationResult(request, driver, pool, device string, adminAccess ...bool) resourceapi.DeviceRequestAllocationResult {
```

Comment:
Is there any reason why we are grabbing only the zeroth entry of this array?

Should we have a test case where the adminAccess differs in the array?

Reply:
This is a quick-and-dirty way of having an optional parameter. There's never a situation where `adminAccess` has more than one entry.

Reply:
Too dirty! I replaced this with a simple boolean argument.